concepts/system-model/index.md

```diff
@@ -10,24 +10,19 @@ An Autonity network is a distributed blockchain system comprised of peer nodes r
 ## Participants
 A participant is a network node connected to an Autonity distributed system and forming part of the system's distributed virtual machine. Participants run a peer node running main client software implementing the [Autonity Protocol](/glossary/#autonity-protocol), i.e. the [Autonity Go Client (AGC)](/glossary/#autonity-go-client-agc) software, communicate with one another over networking protocols in Autonity's communication layer, execute state transitions in the [EVM](/glossary/#ethereum-virtual-machine-evm) Ethereum runtime environment, and have read and write access to the system ledger. Each participant maintains an up-to-date copy of [system state](/glossary/#system-state) in a [ledger object](/concepts/system-model/#the-ledger-object).
 
-Participants are secured and uniquely identified by public key cryptography. Each participant has a public and private keypair ([node key](/concepts/validator/#p2p-node-key)) for signing the messages it broadcasts at the communication layer. This message signature allows cryptographic verification of message sender identity by recipient participants. 
+Participants are secured and uniquely identified by public key cryptography. Each participant node has cryptographic identity from keypairs, see [P2P node keys: `autonityKeys`](/concepts/validator/#p2p-node-keys-autonitykeys). The `autonitykeys` file contains two private keys that are  used to sign messages in the communication layer for transaction (`nodekey`) and consensus (`consensuskey`) gossiping. These message signatures allows cryptographic verification of message sender identity by recipient participants. The corresponding public keys are accessible to other participants and a participant can identify other participants by public key, therefore. The _public node key_ is used to derive participant identifier and network addresses. Namely, a unique network address (the [enode URL](/concepts/validator/#validator-enode-url)), and a _node address_ for the validator (the public key for the private node key).  The _public consensus key_ is used to identify the validator during consensus participation.
```

Comment:
Not a big deal but I feel like this paragraph says multiple times the same thing

Reply:
Yep, less is more. 

I've removed the repetition (Deleting: The corresponding public keys are accessible to other participants and a participant can identify other participants by public key, therefore. The _public node key_ is used to derive participant identifier and network addresses. Namely, a unique network address (the [enode URL](/concepts/validator/#validator-enode-url)), and a _node address_ for the validator (the public key for the private node key).  The _public consensus key_ is used to identify the validator during consensus participation.).

Commit: 02e11361535d5a6bd62ddc2d20100afbc89391d1.